mico-core/src/test/java/io/github/ust/mico/core/ServiceControllerTests.java

```diff
@@ -642,50 +642,90 @@ public void updateServiceUsesExistingInterfaces() throws Exception {
     public void updateServiceIsOnlyAllowedWithoutUsingExistingInterfaces() throws Exception {
         String updatedDescription = "updated description.";
         MicoServiceInterface serviceInterface = new MicoServiceInterface()
-                .setServiceInterfaceName(SERVICE_INTERFACE_NAME)
-                .setDescription(DESCRIPTION);
-        MicoService service = new MicoService()
-                .setShortName(SHORT_NAME)
-                .setVersion(VERSION)
-                .setDescription(DESCRIPTION)
-                .setServiceInterfaces(CollectionUtils.listOf(
-                        serviceInterface
-                ));
-        MicoService updatedService = new MicoService()
-                .setShortName(SHORT_NAME)
-                .setVersion(VERSION)
-                .setDescription(updatedDescription)
-                .setServiceInterfaces(CollectionUtils.listOf(
-                        serviceInterface
-                ));
+            .setServiceInterfaceName(SERVICE_INTERFACE_NAME)
+            .setDescription(DESCRIPTION);
+        MicoService existingService = new MicoService(SHORT_NAME, VERSION, SERVICE_NAME, DESCRIPTION);
+        existingService.setId(ID);
+        existingService.setServiceInterfaces(CollectionUtils.listOf(
+            serviceInterface
+        ));
+        MicoService updatedService = new MicoService(SHORT_NAME, VERSION, SERVICE_NAME, updatedDescription);
+        updatedService.setServiceInterfaces(CollectionUtils.listOf(
+            serviceInterface
+        ));
 
-        given(serviceRepository.findByShortNameAndVersion(SHORT_NAME, VERSION)).willReturn(Optional.of(service));
+        given(serviceRepository.findByShortNameAndVersion(SHORT_NAME, VERSION)).willReturn(Optional.of(existingService));
+
+        StringBuilder urlPathBuilder = new StringBuilder(300);
+        urlPathBuilder.append(SERVICES_PATH);
+        urlPathBuilder.append("/");
+        urlPathBuilder.append(SHORT_NAME);
+        urlPathBuilder.append("/");
+        urlPathBuilder.append(VERSION);
```

Comment:
add shorter variant for creating / concatenating string

Reply:
@buehlefs Is it okay for you, if I remove this path builder stuff? For me it is too verbose for such an simple unit test.